main/main.cpp

```diff
@@ -2088,6 +2088,8 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	GLOBAL_DEF_BASIC(PropertyInfo(Variant::INT, "xr/openxr/view_configuration", PROPERTY_HINT_ENUM, "Mono,Stereo"), "1"); // "Mono,Stereo,Quad,Observer"
 	GLOBAL_DEF_BASIC(PropertyInfo(Variant::INT, "xr/openxr/reference_space", PROPERTY_HINT_ENUM, "Local,Stage"), "1");
 	GLOBAL_DEF_BASIC(PropertyInfo(Variant::INT, "xr/openxr/environment_blend_mode", PROPERTY_HINT_ENUM, "Opaque,Additive,Alpha"), "0");
+	GLOBAL_DEF_BASIC(PropertyInfo(Variant::INT, "xr/openxr/foveation_level", PROPERTY_HINT_ENUM, "Off,Low,Medium,High"), "2");
+	GLOBAL_DEF_BASIC("xr/openxr/foveation_dynamic", true);
```

Comment:
I'm not sure about these as defaults. When I first loaded into my test scene, the dynamic foveated rendering was really unsettling - it seems to turn on and off rapidly. After a little while, it settled down - I'm assuming there was just a performance hit right at the beginning due to shaders needing to compile?

Anyway, I think it might be better for the defaults to be:

- Foveation Level: Off
- Foveation Dynamic: False

What do you think?

Reply:
I'm not against this, we do generally have a principle of being conservative with settings so existing projects don't suddenly change.